spec/models/application_letter_spec.rb

```diff
@@ -99,8 +85,7 @@
   end
 
   it "calculates the correct age of applicant when event starts" do
-    user = FactoryGirl.build(:user)
-    profile = FactoryGirl.build(:profile, user: user)
+    user = FactoryGirl.build(:user_with_profile)
```

Comment:
Yes, I must have accidentally deleted the 2nd line while merging, the tests errored and I fixed it later. It seems cleaner like this, so I would keep it.